server/routes/visits.ts

```diff
@@ -56,8 +56,14 @@ export default function routes(
     }
 
     // VB-1497 - temporary workaround for capacity counts for Hewell / Bristol
-    const maxSlotDefaults =
-      prisonId === 'HEI' ? { OPEN: 30, CLOSED: 3, UNKNOWN: 30 } : { OPEN: 20, CLOSED: 1, UNKNOWN: 20 }
+    let maxSlotDefaults: { OPEN: number; CLOSED: number; UNKNOWN: number }
+    if (prisonId === 'HEI') {
+      maxSlotDefaults = { OPEN: 30, CLOSED: 3, UNKNOWN: 30 }
+    } else if (prisonId === 'BLI') {
+      maxSlotDefaults = { OPEN: 20, CLOSED: 1, UNKNOWN: 20 }
+    } else {
+      maxSlotDefaults = { OPEN: 0, CLOSED: 0, UNKNOWN: 0 }
```

Comment:
Could actually put defaults in the initial `let` to save the `else`? Will approve but would make it tidier 🙂

Reply:
Looks fine - but why do we need a value for UNKNOWN here?

Reply:
> Looks fine - but why do we need a value for UNKNOWN here?

Not sure - haven't looked into this. I think @psoleckimoj added this to handle case where viewing visits in the past might have this visit restriction value?

Reply:
Yep, it all falls over without Unknown as all past visits are unknown.

Reply:
👍 - looks like https://dsdmoj.atlassian.net/browse/VB-1162 has the details

Reply:
sorry - any chance we could make the defaults 2,1,0 or similar - i.e. not live-like but we are able to introduce new prisons for testing etc without having to change code